docs/casa/plugins/cert-authn.md

```
@@ -279,6 +326,30 @@ We chose not to require the presence of the purpose.
 
 ### What's the `roundTripMaxTime` property in the Agama project used for?
 
-The implementation requires defining a maximum time for the completion of the task consisting of initiating enrollment (or authentication), selecting a certificate, entering a PIN (in the case of smart cards), and returning to the page that originated the task. A default value of 30 seconds is used.
+The implementation requires defining a maximum time for the completion of the task consisting of initiating enrollment (or authentication), selecting a certificate, entering a PIN (in the case of smart cards), and returning to the page that originated the task. A default value of 45 seconds is used.
 
 If the process is not completed in this timeframe, it will fail and the log will report errors regarding expired cache entries. In this case, consider using a higher value for `roundTripMaxTime`.
+
+### How does account onboarding works?
+
+Registering an account based on the information found in a certificate is done via attribute mappings:
+Config property `mappingClassField` of flow `io.jans.casa.cert.standaloneOneStepAuthn` points to a Java method found in class `io.jans.casa.certauthn.AttributeMappings` of the cert-authn Agama project. Such method takes a `Map<String, String>` as input and returns a `Map<String, String>` as well. 
```

markdownlint-cli2 (0.20.0)

[warning] Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)